overlord/state/export_test.go

```diff
@@ -32,4 +32,8 @@ func ChangeUnlockCheckpointRetryParamsForTest(newInterval, newMaxTime time.Durat
 	return
 }
 
+func (r *TaskRunner) Handlers() map[string]HandlerFunc {
```

Comment:
this is a bit confusing, it will look in tests too much like is an actual method of TaskRunner, I usually use functional accessors like TaskRunnerHandlers(r)